eng/pipelines/common/xplat-setup.yml

```diff
@@ -21,6 +23,8 @@ jobs:
           value: '.cmd'
         - name: dir
           value: '\'
+        - name: _msbuildCommand
+          value: powershell -ExecutionPolicy ByPass -NoProfile eng\common\msbuild.ps1 -warnaserror:0 -ci
```

Comment:
qq, why is warnaserror set to false here?

Reply:
Actually I don’t remember. Generally we don’t use this, currently only to sendtohelix.proj, which should not emit warnings. So I will remove that in my follow up PR.